src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java

```diff
@@ -58,6 +59,14 @@ public void setAwsProtocolOnlyToFalse() {
         software.aws.rds.jdbc.mysql.Driver.setAcceptAwsProtocolOnly(false);
     }
 
+    @AfterAll
+    public static void cleanup() throws SQLException {
+        Driver test = DriverManager.getDriver("jdbc:mysql://localhost");
```

Comment:
"test" isn't good variable name. It should be something meaningful.

Reply:
Yeap you're right. Will change that.